public/script.js

```diff
@@ -169,17 +205,19 @@ function handle_command(message) {
         if (parts.length < 3)
             return;
         active_poll = new poll(parts[0], true, parts.slice(1));
+        command_handled = true;
     }
     else if (msg.startsWith("!endpoll") && is_admin) {
         if (active_poll !== null)
             active_poll.end_poll();
+        command_handled = true;
     }
-
-    else if (msg.startsWith("!vote")) {
-        if (active_poll === null)
-            return;
+    else if (active_poll !== null && active_poll.is_valid_vote(message.message)) {
         active_poll.handle_vote_message(message);
+        command_handled = true;
     }
+
+    return command_handled;
```

Comment:
Could return `true` here then return `false` instead of `void` in the nested `if` statements. This eliminates the need for `command_handled` and has the benefit of a single return type.

Reply:
Yeah those `void` returns weren't on purpose, I just forgot to make them return false.